server/src/main/java/com/google/musicanalysis/site/AnalysisServlet.java

```diff
@@ -18,13 +18,14 @@
 import javax.servlet.http.HttpServletResponse;
 import java.util.regex.Matcher; 
 import java.util.regex.Pattern; 
+import java.time.Instant;
 
 @WebServlet("/api/analysis")
 public class AnalysisServlet extends HttpServlet {
   @Override
   protected void doGet(HttpServletRequest req, HttpServletResponse res)
       throws ServletException, IOException {
-
+    long ONE_DAY_IN_SECONDS = 86400;
```

Comment:
same as Ibiyemi noted above, spread this number out into the equation that resulted in it